### DISCUSSION TITLE
PK-Sim: Gastric pH and meal

### DISCUSSION THREAD

**Discussion by Erik-Sjogren** [Open-Systems-Pharmacology/Forum#856] [Questions & Problems]
Hi,
I have a couple of PK-Sim suggestions related to the gastric pH and meal event

1. Include Lumen│Stomach│pH as a selectable simulation output (observer) in PK-Sim. This to enable visualization of the dynamics without needing to import an observer for this.
2. Add the display-and-adjust possibility to the parameter (Meal_pH_exponent) that determine the pH dynamics after a meal. Together with the observer (1) this would allow to adjust the pH dynamics according to the nature of simulated meal as seem most appropriate.   
3. Change the current default value of Meal_pH_exponent so to mimic a more realistic time to return to a gastric pH<2.5.
The current setting can have rather large implications for fed state simulations of poorly soluble bases/acids when applying the particle dissolution function.
-The current default value of 1 --> 2 min
-An alternative lower value of 0.01 --> 3.25 h 

I look forward to hearing your thoughts on this.

Many thanks for all great work and efforts.
/Erik

**Comment by AndreDlm** [comment]
This would indeed be very helpful features in PK-Sim and I am fully supportive of it!
Where did you get the alternative values of the Meal_pH_exponent from?

@PavelBal: this might be also relevant for you?

**Comment by StephanSchaller** [comment]
Hi Erik,

if I understand correctly, next to the Observer for luminal pH, the idea would be to have more control about (i.e. customizing) the meal effect in PK-Sim, right?
Or even to be able to add your own customized (pre-defined) meal types/effects?

Ideally, you could add this as a feature request to the PK-Sim repository?

Best,
Stephan

**Comment by Erik-Sjogren** [comment]
@AndreDlm:
Thanks for your support.
The alternative exponent was acquired by manually iteration and visual inspection of the pH-profile.

**Comment by Erik-Sjogren** [comment]
@StephanSchaller 
Yes, correct. 
Absolutely, that would be a nice feature in my mind. I will add it as a request.
Many thanks
